src/index.ts

```diff
@@ -0,0 +1,59 @@
+import { TodoListAppStyles } from './components/todo-list-styles.components';
+import './index.css';
+import TodoListApp from './todo-list/todo-list';
+
+(function () {
+  try {
+    firstRender().render();
+    secondRender().render();
+  } catch (e) {
+    console.error('Error initializing TodoListApp:', e);
+  }
+})();
+
+function firstRender() {
+  const root = document.getElementById('app');
+  const todoListApp = new TodoListApp({
+    el: root as HTMLDivElement,
+    useDnd: true,
+  });
+  return todoListApp;
+}
```

Comment:
```suggestion
(function () {
  try {
    TodoListAppLayout('app-layout-1', true).render();
    TodoListAppLayout('app-layout-2', false).render();
  } catch (e) {
    console.error('Error initializing TodoListApp:', e);
  }
})();

function TodoListAppLayout(appId: string, useDnd: boolean) {
  const root = document.getElementById(`${appId}`);
  const todoListApp = new TodoListApp({
    el: root as HTMLDivElement,
    useDnd,
  });
  return todoListApp;
}
```

이런식으로 함수 호출하고
```
  <div id="app-layout-1"></div>
  <div id="app-layout-2"></div>
```

사용해주면 재사용성이 좋을거같습니다.